scully/utils/serverstuff/waitForServerToBeAvailable.ts

```diff
@@ -1,7 +1,9 @@
-import {ssl} from '../cli-options';
+import {ssl, serverTimeout} from '../cli-options';
 import {scullyConfig} from '../config';
 import {httpGetJson} from '../httpGetJson';
 import {logWarn} from '../log';
+
+const maxTries = serverTimeout !== 0 ? Math.ceil(serverTimeout / 125) : 80;
```

Comment:
@SanderElias How about writing this as 
```
const maxTries = serverTimeout ? Math.ceil(serverTimeout / 125) : 80;
```

Validation will take care of Null's, 0's & Empty's.

Reply:
I prefer to be more explicit and defensive in my code.